src/main/kotlin/gdx/liftoff/data/project/Data.kt

```diff
@@ -38,6 +37,16 @@ data class AdvancedProjectData(
    */
   val androidSdkVersion: String = "35"
 
+  /**
+   * Should match the recommended/tested by the libGDX version.
+   */
+  val androidPluginVersion: String = "8.6.1"
+
+  /**
+   * Will be set to this value if using a Java version of at least 8.
+   */
+  val androidDesugaringLibraryVersion: String = "2.1.5"
```

Comment:
Big thanks for updating the version, by the way! I hate having to track down the latest dependency on the Android docs.